From fea8e283ade82e99c6329c00a8fece054929a92b Mon Sep 17 00:00:00 2001
From: Ryan Zezeski <rpz@joyent.com>
Date: Thu, 22 Sep 2016 18:13:04 +0000
Subject: [PATCH] OS-5675 Upgrade openssl to 1.0.1u Reviewed by: Robert
 Mustacchi <rm@joyent.com> Approved by: Robert Mustacchi <rm@joyent.com>

---
 openssl1x/Makefile              |   4 ++--
 openssl1x/openssl-1.0.1t.tar.gz | Bin 4556447 -> 0 bytes
 openssl1x/openssl-1.0.1u.tar.gz | Bin 0 -> 4567068 bytes
 openssl1x/sunw_prefix.h         |   2 ++
 4 files changed, 4 insertions(+), 2 deletions(-)
 delete mode 100644 openssl1x/openssl-1.0.1t.tar.gz
 create mode 100644 openssl1x/openssl-1.0.1u.tar.gz

diff --git a/openssl1x/Makefile b/openssl1x/Makefile
index be0bc32..e74e7c2 100644
--- a/openssl1x/Makefile
+++ b/openssl1x/Makefile
@@ -19,10 +19,10 @@
 # CDDL HEADER END
 #
 # Copyright (c) 2009, 2010, Oracle and/or its affiliates. All rights reserved.
-# Copyright 2015 Joyent, Inc.
+# Copyright 2016 Joyent, Inc.
 #
 
-VER = openssl-1.0.1t
+VER = openssl-1.0.1u
 LIBVER =	1.0.0
 
 include ../Makefile.defs
diff --git a/openssl1x/openssl-1.0.1t.tar.gz b/openssl1x/openssl-1.0.1t.tar.gz
deleted file mode 100644
index 233d54a..0000000
Binary files a/openssl1x/openssl-1.0.1t.tar.gz and /dev/null differ
diff --git a/openssl1x/openssl-1.0.1u.tar.gz b/openssl1x/openssl-1.0.1u.tar.gz
new file mode 100644
index 0000000..f299acd
Binary files /dev/null and b/openssl1x/openssl-1.0.1u.tar.gz differ
diff --git a/openssl1x/sunw_prefix.h b/openssl1x/sunw_prefix.h
index cbb5ced..9a6375e 100644
--- a/openssl1x/sunw_prefix.h
+++ b/openssl1x/sunw_prefix.h
@@ -1320,6 +1320,8 @@
 #pragma redefine_extname	DSO_set_method sunw_DSO_set_method
 #pragma redefine_extname	DSO_set_name_converter sunw_DSO_set_name_converter
 #pragma redefine_extname	DSO_up_ref sunw_DSO_up_ref
+#pragma redefine_extname	dtls1_clear_received_buffer sunw_dtls1_clear_received_buffer
+#pragma redefine_extname	dtls1_clear_sent_buffer sunw_dtls1_clear_sent_buffer
 #pragma redefine_extname	EDIPARTYNAME_free sunw_EDIPARTYNAME_free
 #pragma redefine_extname	EDIPARTYNAME_it sunw_EDIPARTYNAME_it
 #pragma redefine_extname	EDIPARTYNAME_new sunw_EDIPARTYNAME_new
-- 
2.21.0

